cluster/ci/config/prow/plugins.yaml

```diff
@@ -1131,6 +1131,10 @@ config_updater:
       name: ci-operator-master-configs
       additional_namespaces:
       - ci-stg
+    ci-operator/config/**/*master__*.yaml:
```

Comment:
Let's change the entry above, since `master__*` matches `master__variant`

Reply:
Yup, you are right